.github/workflows/check-projects-comment.yml

```diff
@@ -0,0 +1,63 @@
+name: check-projects-comment
+
+on:
+  workflow_run:
+    workflows: ['check-projects']
+    types:
+      - completed
+
```

Comment:
Add `permissions` to reduce the privilege of GITHUB_TOKEN.

https://docs.github.com/en/actions/security-guides/automatic-token-authentication